cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -209,7 +209,13 @@ Multi-factor authentication (MFA) is by far the best defence against the majorit
 
 The [Multifactor Authentication Cheat Sheet](Multifactor_Authentication_Cheat_Sheet.md) contains further guidance on implementing MFA.
 
-#### Account Lockout
+#### Login Throttling
+
+Login Throttling is a protocol used to prevent an attacker from making too many attempts at guessing a password through normal interactive means, it includes:
+
+- Maximum number of attempts.
+
+- ##### Account Lockout
```

Comment:
Is this meant to be a new header, or a list item after "Maximum number of attempts"?

Reply:
Its a header inside, Login throttling

Reply:
In that case it shouldn't have a "-" before it

Reply:
Its a list item to br treated as new header, as I have used "-" before account lockout, so that it can be treated as a list item and at the same time to be treated as A new headet for other informations.